tests/resources/integration_test_data/bal_library.json

```diff
@@ -0,0 +1,48 @@
+{
+    "entities": {
+        "assetized_child_ref_non_assetized_grandchild/floor": {
+            "versions": [
+                {
+                    "traits": {
+                        "openassetio-mediacreation:content.LocatableContent": {
+                            "location": "file:///${TEST_DATA_ROOT}/assetized_child_ref_non_assetized_grandchild/floor1.usd"
```

Comment:
[fix] I think these have one too many slashes, as `$TEST_DATA_ROOT` is an abs path.

Reply:
Interesting, looks like the `ArDefaultResolver` is pretty tolerant of too many leading `/`s.

Reply:
I think it might be a posix thing, as it works in a shell too.